io.edgehog.devicemanager.Geolocation.json

```diff
@@ -0,0 +1,30 @@
+{
+  "interface_name": "io.edgehog.devicemanager.Geolocation",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "aggregation": "object",
+  "ownership": "device",
+  "description": "Generic Geolocation sampled data.",
+  "doc": "Geolocation allows geolocation sensors to stream location data, such as GPS data. It is usually used in combination with AvailableSensors, which makes API client aware of what sensors are present on devices and what measurement systems are used. sensor_id represents a unique identifier for an individual sensor, and should match sensor_id in AvailableSensors when used in combination.",
```

Comment:
Let's make a better description, the old one sounds weird: `Geolocation allows geolocation sensors`